src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -1236,6 +1351,23 @@ public IEnumerable<string> GetCertificateKeysFromLocalState(TransactionOperation
             }
         }
 
+        public IEnumerable<string> GetCertificateKeysFromCluster(TransactionOperationContext context)
```

Comment:
```suggestion
        public IEnumerable<string> GetCertificateThumbprintFromCluster(TransactionOperationContext context)
```

Change cert key to cert thumprint globally

Reply:
Still need to change this, no?

Reply:
fixed